### PR TITLE
fix: Set all messages in messages.h

### PR DESCRIPTION
**Pull request by EdenComp**
# Description

Added all messages in "messages.h" for visibility

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [ ] I have linked this PR to an issue
- [ ] I have linked this PR to a milestone
- [x] I have linked this PR to a project
- [x] I have tested this code
- [x] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: include/messages.h [hunk 1]
```diff
@@ -7,13 +7,38 @@
 
 #pragma once
 
-static const char *const AMBIGUOUS_INPUT = "Ambiguous input redirect.";
-static const char *const AMBIGUOUS_OUTPUT = "Ambiguous output redirect.";
-static const char *const MISSING_COMMAND = "Invalid null command.";
-static const char *const MISSING_NAME = "Missing name for redirect.";
+typedef const char *const msg_t;
```
Comment (Croos3r) [inline include/messages.h:10]: Grand sensei emil à dis non, donc c'est non

**Review by Croos3r — CHANGES_REQUESTED** [review]

File: include/messages.h (hunk 1 above)
Comment (Croos3r) [reply include/messages.h:10]: ok mais message_t on est pas des geux non plus

Comment (RezaRahemtola) [reply include/messages.h:10]: trop long message_t

**Review by RezaRahemtola — APPROVED** [review]
LGTM, later we will make a messages.h for each type of message, or add them directly in the concerned header file

**Pull request merged**